staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -471,31 +477,60 @@ func TestUpdateNodesInExternalLoadBalancer(t *testing.T) {
 				{Service: newService("s1", "888", v1.ServiceTypeLoadBalancer), Hosts: nodes},
 				{Service: newService("s3", "999", v1.ServiceTypeLoadBalancer), Hosts: nodes},
 			},
+			worker: 5,
 		},
 		{
-			// One service has an external load balancer and one is nil: one call.
+			desc: "One service has an external load balancer and one is nil: one call.",
```

Comment:
How about a testcase where number of services is > number of workers?

Reply:
okay. will add that.